### DISCUSSION TITLE
Enhance the landing page

### DISCUSSION THREAD

**Discussion by everly-gif** [aUnicornDev/tabsMonster#5] [Ideas]
**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is. 
Right now, the landing page is pretty empty , having a webpage with the extension's previews and features would be great

**Describe the solution you'd like**
I would like to take over the landing page's development, creating a minimalistic design and develop it.

**Describe alternatives you've considered**
I could also break the webpage into various sections and create issues for other contributors to contribute and just focus on developing one particular section, rather than single handedly doing all of them . It will be beneficial for the repo.

**Additional context**

**Comment by everly-gif** [comment]
I would like to help in any way ! let me know your suggestions

**Comment by aUnicornDev** [comment]
Hey, Thank you for opening the issue. I have some ideas regarding the sections of the landing page.. How can we connect? Want to move this to Github discussions?

**Comment by everly-gif** [comment]
sure let's do it

**Comment by aUnicornDev** [comment]
## #4 Landing Page
![image](https://user-images.githubusercontent.com/86775297/135705849-3fdece50-23bc-4021-af5c-2deb4e1058d0.png)

For the Hero Section I made a [codepen](https://codepen.io/aunicorndev/pen/qBjBKWO) Just click on the O and move around the mouse to see the effect.

Now to start with, we wanna have the extension available for both Chrome and Firefox.

For the Idea, we wanna have a collection of screenshots from Twitter like in he README section.. I have some tweets bookmarked... and we can find more.

For features, We have the ability to share and on twitter, the tabsMonsterHub where we can view a chart of the number of tabs opened per day, and different themes...

### All these screenshots will be available after the development of these features is complete.

**Reply by everly-gif** [reply]
Sounds good.

**Reply by everly-gif** [reply]
I think I can start with the hero and idea section for now. By when will the development of features be complete? Instead of waiting, we can use mock-ups for now as well . You can later just change the images for the features section.

what do you think?

**Reply by aUnicornDev** [reply]
Yes absolutely.. Mockup images will work perfectly!!

**Reply by everly-gif** [reply]
Right then I'll start the development of the first two sections for now.

**Reply by everly-gif** [reply]
will update you the status soon

**Reply by aUnicornDev** [reply]
I'll creare the Issues and let you know.

Take your time 😊

**Reply by everly-gif** [reply]
Hey I am working on this right? (Design and develop the Hero and Idea section of the Landing page #7 )

**Reply by aUnicornDev** [reply]
Yes

**Comment by everly-gif** [comment]
### Hey need some suggestions @aUnicornDev 
![](https://user-images.githubusercontent.com/77877486/135736142-b5fa28ee-f604-4540-9911-9dff72d12940.jpeg)
![](https://user-images.githubusercontent.com/77877486/135736144-5fbbee29-b685-4e6c-b429-bc2d4de1a0aa.jpeg)
![](https://user-images.githubusercontent.com/77877486/135736146-b09b5e4b-dc62-4b08-b3b3-1513d1f46235.jpeg)

![](https://user-images.githubusercontent.com/77877486/135736148-79d4ab6c-b8f8-4319-8f98-cf44ed7f9eb8.jpeg)

**Reply by aUnicornDev** [reply]
Hey @everly-gif , Sorry for the delay. I was away taking a break for some time...

About the designs, I would prefer the 4th one with two buttons with chrome and Firefox rather than two links below.

**Reply by everly-gif** [reply]
<img src="https://user-images.githubusercontent.com/77877486/136109323-14ab21dc-232b-4245-92eb-c1c54259a68a.png" width=50%>
This should be fine I guess

**Reply by everly-gif** [reply]
Ill make a PR for this, regarding the idea section do you want it like a gallery of tweets? Like what layout are we thinking?

**Reply by aUnicornDev** [reply]
So for the Idea section I was thinking of a 2 column section with a collection of 2-4 screenshots of Twitter stacked unevenly on the right and the Description of the Idea on the left...

Would you be comfortable joining a discord server.. because I miss the notifications here.
Here is the invite in case you wanna join https://discord.gg/bYEucWkj

**Reply by aUnicornDev** [reply]
Hi.. Are you still working on any of this?

**Reply by everly-gif** [reply]
at the moment no, been busy you can open it up for other contributors

**Reply by aUnicornDev** [reply]
No problem, feel free to jump back when you like.. For now I am moving ahed with the Design and development of the Landing page.

**Reply by aUnicornDev** [reply]
@MidouWebDev You can pick it from here... I have some basic structure defined..
